src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestUtilities.cs

```diff
@@ -39,7 +39,12 @@ public static void CheckNoEventSourcesRunning(string message = "")
                     eventSource.Name != "System.Runtime.InteropServices.InteropEventProvider" &&
                     eventSource.Name != "System.Reflection.Runtime.Tracing" &&
                     eventSource.Name != "Microsoft-Windows-DotNETRuntime" &&
-                    eventSource.Name != "System.Runtime"
+                    eventSource.Name != "System.Runtime" &&
+
+                    // These event sources show up when hosted in the VS test runner
+                    eventSource.Name != "System.Net.Sockets" &&
+                    eventSource.Name != "Private.InternalDiagnostics.System.Net.Sockets" &&
+                    eventSource.Name != "TestPlatform"
```

Comment:
We keep growing this list.  Should we instead exclude, say, every event source not declared in this assembly?